docs/_pages/releases.md

```diff
@@ -10,6 +10,7 @@ sidebar:
 ## Unreleased
 
 ### What's New
+* Adding `ThatAreAsync()` and `ThatAreNotAsync()` to `MethodInfoSelector` [#645](https://github.com/fluentassertions/fluentassertions/pull/1725)
```

Comment:
`MethodInfoSelector` is an internal class not really visible to consumers, so maybe we can keep it a bit more functional, like:

> Adding `ThatAreAsync()` and `ThatAreNotAsync()` for filtering in method assertions"

Reply:
Done